### DISCUSSION TITLE
theme icarus can't found

### DISCUSSION THREAD

**Discussion by DanielChungYi** [ppoffice/hexo-theme-icarus#1133] [General]
HI
I did set the theme in _config.yml and clone the repo to place in /theme

But the theme can't be found when deploying
more info is shown at 
https://github.com/DanielChungYi/DanielChungYi/actions/runs/3254451363/jobs/5342719203
![image](https://user-images.githubusercontent.com/31957705/195969730-0771468d-3980-4d4c-a4f6-8b04cc2b4213.png)

There is another question here
While I am running hexo server in local
the theme can be found, but this error appears
![image](https://user-images.githubusercontent.com/31957705/195969852-53876880-ea98-4df1-b669-43ea092b670c.png)

You can clone the repo to reproduce the issue.
Any info will be appreciated.

**Comment by ppoffice** [comment]
You are trying to build a Hexo site using JekyII workflow, which obviously will not work. Please read https://hexo.io/docs/github-pages for deploying your site to GitHub pages.

Also, if you install Icarus via source code instead of NPM, you need to make sure the NPM dependencies of Icarus are also installed. This command may help:

```
npm install --save semver bulma-stylus@0.8.0 hexo-component-inferno@^2.0.0 hexo-renderer-inferno@^0.1.3 inferno@^7.3.3 inferno-create-element@^7.3.3
```
